functions/fish_prompt.fish

```diff
@@ -909,6 +910,20 @@ function __bobthefish_prompt_nvm -S -d 'Display current node version through NVM
     set_color normal
 end
 
+function __bobthefish_prompt_nvm -S -d 'Display current node version through FNM'
```

Comment:
this overrides the previously declared NVM function. if we're going to do it this way we need two functions :)